app/static/js/events/event-dashboard.js

```diff
@@ -0,0 +1,64 @@
+// Function that initializes a DataTable on .tables inside $(element).
```

Comment:
I have updated my repo . also done a bit change in column size . THen I simply did git add , and because of that this whole file is getting added.